### PR TITLE
use 'app.R' for new shiny projects

### PR DESCRIPTION
**Pull request by kevinushey**
This PR:

1. Removes the old `shiny-0.9` resources, as I think we can safely assume that all users are now on Shiny 1.0 or newer,

2. Updates 'New Shiny Project' to create 'app.R', rather than the 'ui.R' / 'server.R' pair.

### CODE REVIEW

**Comment by jjallaire** [comment]
If it's not already the case we should make sure the DependencyManager lists Shiny v1.0 or higher as it's version to install.

File: src/gwt/src/org/rstudio/studio/client/common/dependencies/DependencyManager.java [hunk 1]
```diff
@@ -322,7 +322,7 @@ public void execute(Boolean succeeded)
       deps.add(Dependency.cranPackage("R6", "2.0"));
       deps.add(Dependency.cranPackage("sourcetools", "0.1.5"));
       deps.add(Dependency.cranPackage("htmltools", "0.3.5"));
-      deps.add(Dependency.cranPackage("shiny", "0.13", true));
+      deps.add(Dependency.cranPackage("shiny", "1.0"));
```
Comment (jmcphers) [inline src/gwt/src/org/rstudio/studio/client/common/dependencies/DependencyManager.java]: Why did we formerly require installing shiny as a source package? (don't actually recall, just curious why it's no longer necessary!)

Comment (kevinushey) [reply src/gwt/src/org/rstudio/studio/client/common/dependencies/DependencyManager.java]: I believe that, at the time, binary packages for v0.13 of Shiny were not available on CRAN, so we tried to force installation from sources.

**Comment by kevinushey** [comment]
@jmcphers I'll let you merge this one when you're back in if all looks well.

**Comment by jmcphers** [comment]
LGTM!

**Comment by jjallaire** [comment]
We should always install shiny and rmarkdown as source packages. This is so that they install properly on older versions of R, since CRAN only makes binaries available for the current and previous versions of R.

**Comment by jjallaire** [comment]
So let's make the change back to source == `true` here

**Comment by kevinushey** [comment]
Thanks for the heads up -- I'll make that change.

**Comment by kevinushey** [comment]
https://github.com/rstudio/rstudio/commit/6409fa52fec3ff75ff0e14727775180d97462696

**Comment by jmcphers** [comment]
I'd forgotten that, thanks @jjallaire (and @kevinushey!)

**Pull request merged**